08 - Fun with HTML5 Canvas/index-START.html

```diff
@@ -1,19 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>HTML5 Canvas</title>
-</head>
-<body>
-<canvas id="draw" width="800" height="800"></canvas>
-<script>
-</script>
-
-<style>
-  html, body {
-    margin: 0;
-  }
-</style>
-
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>HTML5 Canvas</title>
+  </head>
+  <body>
+    <canvas id="draw" width="800" height="800"></canvas>
+    <script>
+      const canvas = document.querySelector("#draw");
+      canvas.height = window.innerHeight;
+      canvas.width = window.innerWidth;
+
+      const ctx = canvas.getContext("2d");
+      ctx.lineWidth = 100;
+      ctx.lineCap = "round";
+
+      let painting = false;
+      let hue = 0;
+      let brushWidth = true;
```

Comment:
@ct-aa, I think it's best to practice using descriptive names for a variable, especially for boolean. For example, if I am not mistaken, the purpose of the `brushWidth` variable is to determine whether the brush width is increasing or decreasing, right? We can name it something like `isSizeIncreasing`, or something that best describes the purpose of the variable. Also, the same thing for `painting`.